js/data.js

```diff
@@ -1,19 +1,17 @@
 /* exported data */
-var data = {
+let data = {
```

Comment:
objects and arrays should always be `const` variables because you can update the values they contain since that is not changing what is stored in the variable.